plugins/automation/admin/assets/javascripts/admin/components/fields/da-trust-levels-field.gjs

```diff
@@ -7,16 +9,25 @@ import DAFieldLabel from "./da-field-label";
 export default class TrustLevelsField extends BaseField {
   @service site;
 
+  @action
+  onChangeTrustLevels(value) {
+    if (isBlank(value)) {
+      value = undefined;
+    }
+
+    this.mutValue(value);
+  }
+
   <template>
     <section class="field category-field">
       <div class="control-group">
         <DAFieldLabel @label={{@label}} @field={{@field}} />
 
         <div class="controls">
           <MultiSelect
-            @value={{@field.metadata.value}}
+            @value={{readonly @field.metadata.value}}
```

Comment:
Any specific reason why we're using `readonly` in some fields but not others?

Reply:
We generally don't need it anymore, I just have ptsd of this, it's harmful if not needed anyways. I can remove it here :+1: